src/main/kotlin/net/atos/zac/app/shared/RestPageParameters.kt

```diff
@@ -0,0 +1,10 @@
+package net.atos.zac.app.shared
+
+import jakarta.validation.constraints.Positive
+import nl.lifely.zac.util.NoArgConstructor
+
+@NoArgConstructor
+open class RestPageParameters(
+    @JvmField @Positive var page: Int,
```

Comment:
Do we really need the `@JvmField` here? Normally we don't use them and instead use the getters and setters from Java code to get to the fields. I think that's a bit cleaner maybe?

Reply:
This one is now extended by Java class and there `page` and `rows` are used as static fields.

When we migrate the Java part we can get rid of the `@JvmField`

Reply:
clear!